cafe/templatetags/query_transform.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
```

Comment:
Could you please add a docstring describing why you need this helper function

Reply:
I removed this templatetag because it is not needed.